### PR TITLE
Always add the mixed attribute

### PR DESCRIPTION
**Pull request by sbrunner**
Related to: https://github.com/camptocamp/ngeo/issues/1248

Fix: #2277

@pgiraud @olivierSemet is it ok for you?

### CODE REVIEW

File: c2cgeoportal/tests/functional/test_themes_mixed.py [hunk 1]
```diff
@@ -0,0 +1,265 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2013-2016, Camptocamp SA
```
Comment (pvalsecc) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: Weird to have a 2013-2016 copyright on a new file. Shouldn't that be only 2016?

File: c2cgeoportal/tests/functional/test_themes_mixed.py [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+
+        for t in DBSession.query(UIMetadata).all():
+            DBSession.delete(t)
+        for t in DBSession.query(WMTSDimension).all():
+            DBSession.delete(t)
+        for layer in DBSession.query(Layer).all():
+            DBSession.delete(layer)
+        for layergroup in DBSession.query(LayerGroup).all():
+            DBSession.delete(layergroup)
+        for t in DBSession.query(Theme).all():
```
Comment (pvalsecc) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: For info, this kind of stuff can be done like that:
```python
DBSession.query(UIMetadata).delete()
```

File: c2cgeoportal/tests/functional/test_themes_mixed.py [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+        if "children" in item:
+            result["children"] = [
+                self._only_name(i, attributes) for i in item["children"]
+            ]
+
+        return result
+
+    def _get_filtered_errors(self, themes):
+        prog = re.compile("^The layer '' \(__test_layer_external_wms\) is not defined in WMS capabilities$")
```
Comment (pvalsecc) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: Why a regex?

File: c2cgeoportal/tests/functional/test_themes_mixed.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: Yes :-)

File: c2cgeoportal/tests/functional/test_themes_mixed.py (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: Effectively, thanks :-)

File: c2cgeoportal/tests/functional/test_themes_mixed.py (hunk 3 above)
Comment (sbrunner) [inline c2cgeoportal/tests/functional/test_themes_mixed.py]: Probably history :-)

**Comment by sbrunner** [comment]
All fixed :-)

**Comment by pvalsecc** [comment]
You can merge when Travis is happy.

**Pull request merged**